middlewares/auth.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+
+const validateToken = async (req, res, next) => {
+  try{
+    const secretKey = process.env.SECRET_KEY;
+    const token = req.headers.authorization;
+    const decoded = jwt.verify(token, secretKey);
+    
+    if (!decoded) {
+      throw new Error('INVALID_USER');
+    }
+
+    req.user =decoded.userId;
+
+    return next();
+  } catch (err) {
+    res.status(400).json({ message: 'INVALID_ACCESS_TOKEN'});
```

Comment:
token 관련된 error는 status code로 401을 사용해 주시는 것이 좋습니다.
```suggestion
    res.status(401).json({ message: 'INVALID_ACCESS_TOKEN'});
```